docs/isoSPI.md

```diff
@@ -1,3 +1,109 @@
-# AMDC isoSPI Interface
+# AMDC isoSPI and Differential I/O Interface
 
-This document describes the design considerations and implementation details for the isoSPI interface subsystem on the Advanced Motor Drive Controller (AMDC).
+This document describes the design considerations and implementation details for the isoSPI and differential I/O interface on the Advanced Motor Drive Controller (AMDC). The block diagram of the implementation is presented and each block is discussed in detail. Details on the PCB layout, and connector pinout are also provided in this document.
+
+
+## Relevant Versions of AMDC Hardware
+
+AMDC REV D
+
+
+## Design Requirements and Considerations
+
+The isolated serial peripheral interface (isoSPI) has increased noise immunity and safety. The following describes the design requirements for the isoSPI and differential I/O (D_IO) interface. The design requirements for the isoSPI and differential I/O (D_IO) interface are as follows:

```

Comment:
> The isolated serial peripheral interface (isoSPI) has increased noise immunity and safety.

I don't know what to make of this sentence. What information is it trying to convey to the reader?

Reply:
I have updated this section.